homeassistant/components/bluetooth/manager.py

```diff
@@ -225,15 +225,18 @@ def async_get_scanner_discovered_devices_and_advertisement_data_by_address(
         results: list[tuple[BaseHaScanner, BLEDevice, AdvertisementData]] = []
         for type_ in types_:
             for scanner in self._get_scanners_by_type(type_):
-                if device_advertisement_data := scanner.discovered_devices_and_advertisement_data.get(
+                if adv_data := scanner.discovered_devices_and_advertisement_data.get(
```

Comment:
This is a tuple of ble_device and adv_data.

Reply:
Right, how would one shorten that? 88 is the max line length

Reply:
Maybe split that line and make a new var for scanner.disco.....

Reply:
I could not find a viable way that in some way looked normal, so feel free to suggest an improvement.

Reply:
Will take a look when I'm back at my desk, suggesting on mobile is too painful 😬

Reply:
Suggestions inlined below